### ISSUE TITLE
[Submission] Icon name

### ISSUE THREAD

**Issue by cedricholz** [Shopify/polaris-icons#678]
Unlocked

**Comment by kaelig** [comment]
@cedricholz can you follow the issue template provided? Thank you.

**Comment by cedricholz** [comment]
I clicked this from https://github.com/Shopify/polaris-icons.

![image](https://user-images.githubusercontent.com/3516769/65648327-b7ab2780-dfb6-11e9-81fe-ba1f1ac4620a.png)

And it led me to a blank issue.

![image](https://user-images.githubusercontent.com/3516769/65648348-cc87bb00-dfb6-11e9-8450-c2024d9c096e.png)

I see the template when I click "New Issue" on github or click "Create a new version of this icon" from the https://polaris-icons.shopify.com/

We don't actually need this any more as we created our own, I just thought someone else might want an unlock for the lock button.
